axelrod/strategies/rand.py

```diff
@@ -10,3 +12,51 @@ class Random(Player):
 
     def strategy(self, opponent):
         return random.choice(['C', 'D'])
+
+
+class Tullock(Player):
+    """
+    Cooperates for the first 11 rounds then randomly cooperates 10% less often
+    than the opponent has in previous rounds."""
+
+    name = "Tullock"
+    memoryone = False # memory-10
+
+    def strategy(self, opponent):
+        if len(self.history) < 11:
+            return 'C'
+        counter = Counter(opponent.history[-10:])
+        prop_cooperate = counter['C'] / 10.
+        prob_cooperate = max(0, prop_cooperate - 0.10)
+        r = random.random()
+        if r < prop_cooperate:
+            return 'C'
+        return 'D'
+
+class Feld(Player):
+    """
+    Defects when opponent defects. Cooperates with a probability that decreases
+    to 0.5 at round 200.
+    """
+
+    name = "Feld"
+    memoryone = False # Probabilities are not constant
+
+    def _cooperation_probability(self):
+        """It's not clear what the iterpolating function is, so we'll do
+        something simple that decreases monotonically from 1.0 to 0.5 over
+        200 rounds."""
+        slope = 0.5/200
```

Comment:
I've been wondering about the '200'... There are other strategies that do things up until K rounds from the end. Could be worth having the number of rounds as property that the players can access... In this particular instance I think it should be a parameter.... The idea of the players being able to access the number of rounds could be another PR perhaps...